src/components/IssueQueue.vue

```diff
@@ -52,7 +56,16 @@ const params = computed(() => {
   const paramsObj: Record<string, any> = {};
 
   if (isMyIssuesSelected.value) {
-    paramsObj.owner = userStore.userName;
+    if (userStore.jiraUsername === '') {
+      addToast({
```

Comment:
This message is already part of the `cacheJiraUsername` function that is called after login, this could lead to a double notification